recipe/meta.yaml

```diff
@@ -6,20 +6,21 @@ package:
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: 5731e09d5f82c7f8e3849b12ec9c7dd0b6afbab1a6c70842e99e5d4478954edb
 
 build:
   noarch: python
   script: {{ PYTHON }} -m pip install . -vv
-  number: 0
+  number: 1
 
 requirements:
   host:
-    - python >=3.8
+    - python {{ 3.8 }}
```

Comment:
What the linter asks you to do is to write this as `python {{ python_min }}` (same below, modulo `>=`). You will need to rerender again afterwards for this to work.